lp-to-jira

```diff
@@ -27,11 +29,141 @@ pkg_to_component = {
     "openjdk-lts": "OpenJDK",
     "apt": "Package Management",
     "aptdaemon": "Package Management",
-    "ubuntu-release-upgrader": "Package Management"
+    "ubuntu-release-upgrader": "Package Management",
+    "solutions-qa-ci": "Solutions QA CI",
+    "cpe-foundation": "FCE",
 }
 default_component = "Distro"
 
 
+def getLpBug(lp, bug_number):
+    """Make sure the bug ID exists, return bug"""
+
+    bug = None
+    try:
+        bug = lp.bugs[bug_number]
+    except Exception:
+        print("Couldn't find the Launchpad bug {}".format(bug_number))
+        sys.exit(1)
+
+    return bug
+
+
+def getLpBugPkg(lp, bug):
+    """From a LP bug, get its package"""
+
+    bug_pkg = None
+    if len(bug.bug_tasks) == 1:
+        bug_pkg = bug.bug_tasks[0].bug_target_name.split()[0]
+    else:
+        for task in bug.bug_tasks:
+            if "(Ubuntu)" in task.bug_target_name:
+                bug_pkg = task.bug_target_name.split()[0]
+
+    return bug_pkg
+
+
+def getAllLpProjectBugs(lp, project, days=None):
+    """Return iterable IBugTasks of all Bugs in a Project filed in the past n
+    days. If days is not specified, return all Bugs."""
+
+    project = lp.projects[project]
```

Comment:
You may need to catch an exception if the project doesn't exist.
I would also use different variable name for local project and function param project (unless this is some pythonic thing I'm not yet familiar with)

Reply:
LP will throw a 404 if the project doesn't exist, but I'll catch it to make a friendlier print.
And that variable name `project` is totally nothing pythonic. Good catch!